modules/virt-defining-a-watchdog.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog"]
+= Defining a watchdog device
+
+.Procedure
+
+. Choose the watchdog action. This defines how the watchdog proceeds when the operating system (OS) no longer responds.
+
++
+.Available actions
+[horizontal]
+`poweroff`:: The virtual machine (VM) powers down immediately.
+`reset`:: The VM reboots in place, and the guest OS cannot react. Because the length of time required for the guest OS to reboot can cause liveness probes to timeout, you should avoid their use.
+`shutdown`:: The VM gracefully powers down by stopping all services.
+
+. Create a YAML file and apply it to the cluster.
```

Comment:
Please tell the user how to do this. The user needs to create a yaml file, save it, and then use the 'oc apply' command to update the virtual machine CR.